lib/autobox/Core.pm

```diff
@@ -1858,5 +1924,16 @@ sub ref      { CORE::ref   $_[0] }
 
 sub curry  { my $code = CORE::shift; my @args = @_; sub { CORE::unshift @_, @args; goto &$code; }; }
 
-1;
+##############################################################################################
+#
+# UNDEF
+#
+
+package autobox::Core::UNDEF;
+
+sub defined { 0 }
 
+# make any other method a no-op
```

Comment:
I don't think this is a good idea. Probably best to raise/discuss it in a separate issue.